migrator/replica/rocksdb/db_replica_manager.go

```diff
@@ -0,0 +1,54 @@
+package rocksdb
+
+import (
+	"regexp"
+
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/migrations"
+	"github.com/stackrox/rox/pkg/set"
+)
+
+const (
+	// Indexes
+	bleveIndex = "scorch.bleve"
+	index      = "index"
+
+	// CurrentReplica - active rocksdb replica name
+	CurrentReplica = migrations.Current
+
+	// RestoreReplica - restore rocksdb replica name
+	RestoreReplica = ".restore"
+
+	// BackupReplica - backup rocksdb replica name
+	BackupReplica = ".backup"
+
+	// PreviousReplica - previous rocksdb replica used for rollback
+	PreviousReplica = ".previous"
+
+	// TempReplica - temp rocksdb replica
+	TempReplica = "temp"
+
+	// EmergencyReplica - emergency rocksdb replica, only to be used in the event an
+	// upgrade to Postgres was followed with a rollback
+	EmergencyReplica = ".emergency"
```

Comment:
No, it wont work.
The earlier version wont be able to recognize the new EmergencyReplica. 
We can use the .previous link for rollback. I feel like we do not need to create a new emergency replica here. After migration, we should have .previous and a temp replica in preparing for migration. We can use .previous replica for rollback and make the temp replica as a fake current which can be recognized by Postgres.

Reply:
you are right, it won't work.  The permutations here are tricky.  Not having a current is not a legal state, currently.  I have an idea, I'm going to work through.

Reply:
The more I think about the more I think we are over-complicating that scenario.  If Postgres has been updated based on Rocks then Postgres will have a version >= Rocks.  When we run the migrations we will have to use that knowledge to determine which set of migrations to run.  I don't think it matters if we look at the Rocks replicas and see a current once the initial move to Postgres occurs.  The only time that might be a problem is if we rollback from Postgres to Rocks.  Then when we upgrade again to the same version as before, then we will think that Postgres is newer.  However, we have ROX-12094 to cleanup Postgres residue in the event of a rollback to Rocks.  This is feeling like an edge case that I'm not seeing a solution to right now that doesn't involve human intervention either to clean things up or require them to upgrade beyond the Postgres version.

Reply:
I think I have this covered now.  We can use the timestamp on current to tell us if Rocks or Postgres is the functional DB on the upgrade.  So even if there is residue in Postgres, we should be covered until we figure out how to solve ROX-12094